build/azure-pipelines.yml

```diff
@@ -114,11 +97,11 @@ stages:
               artifactName: build_output
 
   - stage: Build_Docs
-    condition: and(succeeded(), or(startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), ${{parameters.buildApiDocs}}))
+    condition: and(succeeded(), or(eq(stageDependencies.Build.A.outputs['build.NBGV_PublicRelease'], 'True'), ${{parameters.buildApiDocs}}))
     displayName: Prepare API Documentation
     dependsOn: Build
     variables:
-      umbracoMajorVersion: $[ stageDependencies.Build.A.outputs['determineMajorVersion.majorVersion'] ]
+      umbracoMajorVersion: $[ stageDependencies.Build.A.outputs['build.NBGV_VersionMajor'] ]
```

Comment:
Cool to see that tool working. I noticed that the old variable `determineMajorVersion.majorVersion` prefixes a `v` character making the value something like `v12` and so forth. Does the new variable `build.NBGV_VersionMajor` also contain this prefix?

We use this further down when deploying the docs so they get that folder name, e.g. it will directly affect the link to the docs, here's an example: https://apidocs.umbraco.com/v12/csharp/ where the "v12" part is the majorVersion.

On the v14 branch, we expand on this to also change the paths in Storybook just before deploying it for the JavaScript docs.

Reply:
Just tested more thoroughly and I discovered that we add this `v` character manually when publishing, so never mind 😎